integration/tools/validation/src/main/java/alluxio/fuse/correctness/Options.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.fuse.correctness;
+
+import com.google.common.base.Preconditions;
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.ParseException;
+
+/**
+ * This class stores the options passed in from CLI.
+ */
+public final class Options {
+  public static final CommandLineParser PARSER = new DefaultParser();
+  private final String mLocalDir;
+  private final String mFuseDir;
+  private final IOOperations mOperation;
+  private final int mNumThreads;
+  private final int mNumFiles;
+
+  /**
+   * Constructs a {@CorrectnessValidationOptions} object through CLI.
+   *
+   * @param args arguments from CLI
+   * @return a {@CorrectnessValidationOptions} holding all the test options
+   */
+  public static Options createOptions(String[] args) {
+    CommandLine cli;
+    try {
+      cli = PARSER.parse(CorrectnessOptionsParser.OPTIONS, args);
+    } catch (ParseException e) {
+      throw new RuntimeException("Error parsing command line input.", e);
+    }
+    String localDir = cli.getOptionValue(CorrectnessOptionsParser.LOCAL_DIR_OPTION_NAME);
```

Comment:
Let's rename LOCAL_DIR to tmp dir based on previous discussion

Reply:
We should make `/tmp` the default testing directory, but if users want to set this options, `--tmp-dir` in my opinion doesn't look intuitive. What about naming it to "baseline-dir"?

Reply:
more like a tmp dir to store the source data for comparison, doesn't feel like a baseline-dir which i will think it sets the baseline which hard to define what's the baseline